code/modules/hydroponics/prehistoric_plants.dm

```diff
@@ -66,7 +68,7 @@
 	mutants = null
 	packet_icon = "seed-jurlmah"
 	plant_icon = "jurlmah"
-	chems = list(NUTRIMENT = list(1,10),SEROTROTIUM = list(0,10))
+	chems = list(CLONEXADONE = list(1,10))
```

Comment:
@Kurfursten why were these changes made?

Reply:
Now it has been two years, but I think I was thinking having a plant that just produces as much clonex as you want so you can easily generate tons of meat for the kitchen would be much more useful to botanists than some bay psych chem.